test/Interop/Cxx/namespace/class-inline-namespace-irgen.swift

```diff
@@ -0,0 +1,100 @@
+// RUN: rm -rf %t
+// RUN: split-file %s %t
+// RUN: %target-swift-emit-ir -I %t/Inputs -enable-cxx-interop %t/test.swift | %FileCheck %t/test.swift
+
+
+//--- Inputs/module.modulemap
+module std {
+  header "string.h"
+  requires cplusplus
+}
+//--- Inputs/string.h
+namespace std { inline namespace __1 {
+
+template<class CharT>
+struct basic_string;
+
+typedef basic_string<char> string;
+
+template<class CharT>
+struct basic_string {
+  void *p;
+
+  const CharT *c_str();
+};
```

Comment:
This doesn't seem like a really necessary part of the test. It's sort of complicated. It might be better to just use simple types/functions.